package.json

```diff
@@ -13,17 +13,16 @@
     "express": "4.15.x",
     "gulp-concat": "2.4.x",
     "gulp-eslint": "2.0.x",
-    "gulp-imagemin": "3.1.x",
     "gulp-istanbul-enforcer": "1.0.x",
     "gulp-less": "3.0.x",
     "gulp-ng-annotate": "0.5.x",
     "gulp-rename": "1.2.x",
     "gulp-sourcemaps": "1.6.x",
     "gulp-uglify": "1.0.x",
     "gulp-util": "3.0.x",
-    "karma": "0.12.x",
     "run-sequence": "1.2.x",
-    "superagent": "3.6.x"
+    "superagent": "3.6.x",
+    "karma": "1.7.x"
```

Comment:
is karma not a dev dependancy?

Reply:
Not sure, I haven't added it but rather updated the version after discussing with Lawrie as an old sub dependency was causing the build to fail. I assumed it was required by the build ran by Ramil's deployment process which doesn't pull devdependencies but I'm checking with Lawrie.

Reply:
Lawrie isn't sure of a reason. I'd be inclined to leave with the update and then have a separate task (ticket) to check if required and move to dev dependencies if ok to do so. Wary of it causing an issue for Ramil when building and not being sure if it's this change or the other changes.